.ado/azure-pipelines.publish.yml

```diff
@@ -28,11 +28,11 @@ steps:
     displayName: Git Authentication
   - template: templates/yarn-install.yml
 
-  - script: |
-      yarn test
-    displayName: 'yarn test'
-  - script: |
-      yarn publish:beachball -- -b origin/master -n $(npmAuth) --access public -y
-    displayName: 'Publish NPM Packages'
+  # - script: |
+  #     yarn test
+  #   displayName: 'yarn test'
+  # - script: |
+  #     yarn publish:beachball -- -b origin/master -n $(npmAuth) --access public -y
+  #   displayName: 'Publish NPM Packages'
```

Comment:
Had this commentd out so I could test without messing with the npm packages. Reverting now that I have